scripts/asm-installer/install_asm

```diff
@@ -2037,6 +2043,43 @@ register_gce_identity_provider() {
   retry 3 kubectl apply -f asm/identity-provider/googleidp.yaml
 }
 
+should_enable_service_mesh_feature() {
+  if [[ "${USE_VM}" -eq 0 ]]; then
+    false
+  fi
+}
+
+enable_service_mesh_feature() {
+  info "Enabling the service mesh feature..."
+
+  retry 2 run curl -s -H "Content-Type: application/json" \
```

Comment:
Please add a comment with the IAM requirement "gkehub.features.create"

Reply:
Done.